.github/workflows/ci_tests.yml

```diff
@@ -31,13 +31,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [ubuntu-latest]
+        # TODO: platform: [ubuntu-24.04, macos-15]
+        platform: [ubuntu-24.04]
         compiler:
-          - cpp: g++
-            c: gcc
-          - cpp: clang++
-            c: clang
-        cpp_version: [17, 20, 23, 26]
```

Comment:
Exemplar officially support C++ 17 with the intention to explore the possibilities to support earlier versions. 

Please keep the lowest matrix version 17.

Reply:
Wow will you develop code for C++26 if it should compile with c++17?

Reply:
done